R/pev.R

```diff
@@ -27,13 +27,13 @@ create_rtss_epi_process <- function(
     }
     
     to_vaccinate <- variables$birth$get_index_of(
-      set = timestep - parameters$rtss_epi_age
+      set = timestep - parameters$pev_epi_age
     )
-    if (parameters$rtss_epi_min_wait == 0) {
+    if (parameters$pev_epi_min_wait == 0) {
       target <- to_vaccinate$to_vector()
     } else {
-      not_recently_vaccinated <- variables$rtss_vaccinated$get_index_of(
-        a = max(timestep - parameters$rtss_epi_min_wait, 0),
+      not_recently_vaccinated <- variables$vaccinated_timestep$get_index_of(
+        a = max(timestep - parameters$pev_epi_min_wait, 0),
```

Comment:
The following was failing for me, I think in relation to this section:
```
p <- get_parameters(overrides = list(
  human_population = 1000,
  individual_mosquitoes = FALSE
))

p <- p |>
  set_pev_epi(
    profile = rtss_profile,
    coverages = c(0.99, 0.99, 0.99),
    timesteps = 365 * (2:4) + 1,
    age = 5 * 365,
    min_wait = 180,
    booster_profile = list(rtss_profile, rtss_profile, rtss_profile),
    booster_timestep = c(190, 190, 190),
    booster_coverage = c(0.8, 0.8, 0.8)
  )

s <- run_simulation(timesteps = 365 *  7, parameters = p)
```

Setting `min_wait = 0` resolved the issue. Is the a reasonable check we can make on what the user inputs here?